webapp/components/features/FollowList/FollowList.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+  <base-card class="follow-list">
+    <template v-if="connections && connections.length">
+      <ds-text tag="h5" color="soft" class="spacer-x-small">
+        {{ userName | truncate(15) }} {{ $t(`profile.network.${type}`) }}
+      </ds-text>
+      <div :class="connectionsClass">
+        <user-teaser
+          v-for="connection in filteredConnections"
+          class="spacer-x-small"
+          :user="connection"
+          :key="connection.id"
+        />
+      </div>
+      <base-button
+        v-if="allConnectionsCount - connections.length"
+        @click="$emit('fetchAllConnections', type)"
```

Comment:
we have been in the habit of putting the `events` at the bottom of the list.

Reply:
I bumped into that question, :smile: 
Just apply [Vue's styleguide regarding attribute order](https://vuejs.org/v2/style-guide/#Element-attribute-order-recommended)?

Reply:
yes, please!

Reply:
done in 8e98570